shipyard_blueprints/hubspot/shipyard_hubspot/hubspot.py

```diff
@@ -85,3 +102,184 @@ def connect(self):
         else:
             self.logger.info("Successfully connected to Hubspot API")
             return 0
+
+    def export_data(self, export_type: str, **kwargs):
+        export_type = validate_export_type(export_type)
+        self.logger.debug("Exporting data from Hubspot API")
```

Comment:
Can you provide some doc strings as to what this method does and what are valid options for the `export_type` as well as valid key word args?